docs_user/modules/proc_adopting-the-object-storage-service.adoc

```diff
@@ -1,26 +1,25 @@
 [id="adopting-the-object-storage-service_{context}"]
 
-= Adopting the Object Storage service
+= Adopting the {object_storage}
 
-This section only applies if you are using OpenStack Swift as {object_storage_first_ref}. If you are using the Object Storage API of Ceph RGW this section can be skipped.
+If you are using OpenStack Swift as an {object_storage}, adopt the {object_storage} to the {rhos_long} environment. If you are using the Object Storage API of the Ceph Object Gateway (RGW), skip this procedure.
```

Comment:
You could maybe put "Optional" at the start of this sentence. Is the point of this line that you have to adopt the {object_storage} to the {rhos_long} environment if you use OpenStack Swift as an {object_storage}? If so, maybe: "If you use OpenStack Swift as an {object_storage}, ensure that you adopt the {object_storage} to the {rhos_long} environment."

Skip what procedure? The "adopt the {object_storage} to the {rhos_long} environment" or what comes next? The original draft makes it look like it's what follows, whereas the rewrite could be interpreted as the step you just mentioned in the previous line. Could clarify with "skip the following procedure" or "you do not need to adopt the {object_storage} to the {rhos_long} environment." or whatever the case may be.

Should you say the OpenStack Object Store Service (Swift) or something like that? 

Also maybe OpenStack could be made into a variable so it is OpenStack upstream and RHOSP downstream